### PR TITLE
Broke out IPSec and deprecated here.

### PR DESCRIPTION
**Pull request by keithjjones**
The new code is now here: https://github.com/corelight/zeek-spicy-ipsec

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Currently, the following analyzers are included:
 - DNS <sup>[1]</sup>
 - [Facefish Rootkit](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/facefish_rootkit)
 - HTTP <sup>[1]</sup>
-- [IPSec](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/ipsec)
+- IPSec deprecated by [zeek-spicy-ipsec](https://github.com/corelight/zeek-spicy-ipsec)
```
Comment (bbannier) [inline README.md:14]:
```suggestion
- IPSec from [zeek-spicy-ipsec](https://github.com/corelight/zeek-spicy-ipsec)
```

File: analyzer/CMakeLists.txt [hunk 2]
```diff
@@ -6,7 +6,6 @@ add_subdirectory(dhcp)
 add_subdirectory(dns)
 add_subdirectory(facefish_rootkit)
 add_subdirectory(http)
-add_subdirectory(ipsec)
```
Comment (bbannier) [inline analyzer/CMakeLists.txt:9]: Just piggy-backing on this change which looks good:

Let's instead pull this package in via a dependency in `zkg.meta`. Please add the dependency with a specific version tag, e.g.,

```
depends = zeek-spicy-ipsec ==0.2.1
```

That way users of `spicy-analyzers` keep seeing the same analyzers.

**Review by bbannier — CHANGES_REQUESTED** [review]

**Comment by keithjjones** [comment]
Cool, will do.  I'll fix this up after the holiday.  Thanks!

File: analyzer/CMakeLists.txt (hunk 2 above)
Comment (keithjjones) [reply analyzer/CMakeLists.txt:9]: Done.  Please double check me.  Thanks!

**Comment by keithjjones** [comment]
This is now in:  https://github.com/zeek/spicy-analyzers/pull/107

**Pull request closed without merging**